styles/desktop.css

```diff
@@ -0,0 +1,271 @@
+body {
+  margin: 0;
+  display: flex;
+  justify-content: center;
+  flex-flow: row wrap;
+  font-family: Helvetica, sans-serif;
+}
+header {
+  background-color: #213d69;
+  padding: 2rem 6rem 3.5rem 6rem;
+  width: 100%;
+}
+footer {
+  background-color: #474b52;
+  padding: 3rem 0 4rem 0;
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  margin-top: 4rem;
+}
+nav {
+  width: 100%;
+}
+h1 {
+  color: #4eba62;
+  font-family: "Times", serif;
+  font-style: italic;
+  font-weight: normal;
+  font-size: 1.6rem;
+  margin: 0;
+}
+h2 {
+  font-family: Georgia, sans-serif;
+  color: #213d69;
+  font-size: 2.5rem;
+  margin: 0rem 0 1rem 0;
+}
+h3 {
+  font-family: Georgia, sans-serif;
+  color: #213d69;
+  font-size: 1.3rem;
+}
+h4 {
+  font-family: Georgia, sans-serif;
+  color: #213d69;
+  font-size: 1.9rem;
+  margin: 0 0 0.83rem 0;
+}
+h5 {
+  font-family: Georgia, sans-serif;
+  color: #213d69;
+  font-size: 1.5rem;
+  margin: 2rem 0 0.4rem 0;
+}
+h6 {
+  font-size: 0;
```

Comment:
Why the font size 0?

Reply:
Sorry for late response, but this header is for screenreader only.